Fix trigger test

elias-ba opened the pull request:
## Notes for the reviewer

## Related issue

Fixes #

## Review checklist

- [ ] I have performed a **self-review** of my code
- [ ] I have verified that all appropriate **authorization policies** have been implemented and tested
- [ ] If needed, I have updated the **changelog**
- [ ] Product has **QA'd** this feature

taylordowns2000 reviewed (changes requested):
there are 14 occurrences of the word "credential" in the `trigger_test` and I don't think any of them actually refer to a credential. please change all of them so they're named appropriately (e.g., `credential_name = "funnycredentialname"` becomes `auth_method_name = "funnyauthmethodname"`)

then could you have a quick think/search to see if there's any more bad naming going on related to webhook authentication methods? we had a bit of a snafu early on (`webhook_authentication_methods` are _NOT_ `credentials` and shouldn't be called credentials in the code) and we've tried to clean it up but clearly missed a few spots!

thank you!!!

elias-ba commented:
> there are 14 occurrences of the word "credential" in the `trigger_test` and I don't think any of them actually refer to a credential. please change all of them so they're named appropriately (e.g., `credential_name = "funnycredentialname"` becomes `auth_method_name = "funnyauthmethodname"`)
> 
> then could you have a quick think/search to see if there's any more bad naming going on related to webhook authentication methods? we had a bit of a snafu early on (`webhook_authentication_methods` are _NOT_ `credentials` and shouldn't be called credentials in the code) and we've tried to clean it up but clearly missed a few spots!
> 
> thank you!!!

@taylordowns2000 thanks for this change request. I have updated the PR. There are many occurences of the word 'credential' in the code but I believe all of them referencing webhook auth methods are now changed.
